app.rb

```diff
@@ -0,0 +1,166 @@
+require_relative 'person'
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'rental'
+require_relative 'book'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  # Show menu for users
+  def show_menu
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts "7 - Exit\n\n"
+    gets.chomp
+  end
+
+  # Method that handles user choice
+  def run
+    user_choice = show_menu
+    case user_choice
+    when '1'
+      action_list_books
+    when '2'
+      action_list_people
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      list_rental
```

Comment:
- [ ] The method you have for listing rentals is `list_rentals` plural instead of `list_rental`. Besides, the methods takes in person id as parameter. Therefore, you may refactor this line to something like the following to be able to display list of rentals by an individual.
```
puts 'Enter person\'s ID number:'
list_rentals(gets.chomp.to_i)
```
![image](https://user-images.githubusercontent.com/44057863/175788052-14115ef1-c1ba-4abd-bc4f-481ce695062b.png)